content/help/ops/setup/validation/index.md

```diff
@@ -0,0 +1,320 @@
+the data from this configmap to create and update the
+`validatingwebhookconfiguration`.
+
+{{< text bash yaml >}}
+$ kubectl -n istio-system get configmap istio-galley-configuration -o jsonpath='{.data}'
+map[validatingwebhookconfiguration.yaml:apiVersion: admissionregistration.k8s.io/v1beta1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: istio-galley
+  namespace: istio-system
+  labels:
+    app: istio-galley
+    chart: galley-1.0.0
+    release: istio
+    heritage: Tiller
+webhooks:
+  - name: pilot.validation.istio.io
+    clientConfig:
+      service:
+        name: istio-galley
+        namespace: istio-system
+        path: "/admitpilot"
+      caBundle: ""
+    rules:
+      - operations:
+      (... snip ...)
+{{< /text >}}
+
+If the webhook array in `istio-galley-configuration` is empty and
+you're using `helm template` or `helm install`, verify `--set
+galley.enabled` and `--set global.configValidation=true` options are
+set. If you're not using helm, you'll need to find a generate
+YAML that includes the populated webhook array.
+
+The `istio-galley` validation configuration is fail-close. If
+configuration exists and is scoped properly, the webhook will be
+invoked. A missing `caBundle`, bad certificate, or network connectivity
+problem will produce an error message when the resource is
+created/updated. If you don’t see any error message and the webhook
+wasn’t invoked and the webhook configuration is valid, your cluster is
+misconfigured.
+
+## Creating configuration fails with x509 certificate errors
+
+`x509: certificate signed by unknown authority` related errors are
+typically caused by an empty `caBundle` in the webhook
+configuration. Verify that it is not empty (see [verify webhook
+configuration](#invalid-configuration-is-accepted)). The
+`istio-galley` deployment consciously reconciles webhook configuration
+used the `istio-galley-configuration` `configmap` and root certificate
+mounted from `istio.istio-galley-service-account` secret in the
+`istio-system` namespace.
+
+1. Verify the `istio-galley` pods(s) are running:
+
+    {{< text bash >}}
+    $  kubectl -n istio-system get pod -listio=galley
+    NAME                            READY     STATUS    RESTARTS   AGE
+    istio-galley-5dbbbdb746-d676g   1/1       Running   0          2d
+    {{< /text >}}
+
+1. Verify you’re using Istio version >= 1.0.0. Older version of galley
+   did not properly re-patch the `caBundle`. This typically happened
+   when the `istio.yaml` was re-applied, overwriting a previously
+   patched `caBundle`.
+
+    {{< text bash >}}
+    $ for pod in $(kubectl -n istio-system get pod -listio=galley -o jsonpath='{.items[*].metadata.name}'); do \
+        kubectl -n istio-system exec ${pod} -it /usr/local/bin/galley version| grep ^Version; \
+    done
+    Version: 1.0.0
+    {{< /text >}}
+
+    1. Check the galley pod logs for errors. Failing to patch the `caBundle`
```

Comment:
I think this block should be outdented. As it is, it's creating a sublist with a single entry in it.